src/routes/safe/components/Settings/ManageOwners/AddOwnerModal/index.tsx

```diff
@@ -98,7 +101,7 @@ export const AddOwnerModal = ({ isOpen, onClose }: Props): React.ReactElement =>
     onClose()
 
     try {
-      await sendAddOwner(values, safeAddress, safeVersion, txParameters, dispatch)
+      await sendAddOwner(values, safeAddress, txParameters, dispatch, connectedWalletAddress)
```

Comment:
Is this the only instance of `sendAddOwner`? Just to be double safe because you removed/added arguments at different indexes.

Reply:
If it's only here then I think it's fine.